model/distro/distro_test.go

```diff
@@ -428,3 +429,31 @@ func TestGetResolvedPlannerSettings(t *testing.T) {
 	assert.EqualValues(t, 0, resolved2.TimeInQueueFactor)
 	assert.EqualValues(t, 0, resolved2.ExpectedRuntimeFactor)
 }
+
+func TestAddPermissions(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(db.ClearCollections(user.Collection, Collection, evergreen.ScopeCollection, evergreen.RoleCollection))
+	_ = evergreen.GetEnvironment().DB().RunCommand(nil, map[string]string{"create": evergreen.ScopeCollection})
+	u := user.DBUser{
+		Id: "me",
+	}
+	assert.NoError(u.Insert())
```

Comment:
my inclination is to make these kinds of assertions requires, because you don't want to keep running the test in these cases.

Reply:
fixed